### PR TITLE
Feature: SYNC content to office commit [d3ace3059c4a92ad4b14b3cd05b566…

### PR DESCRIPTION
**Pull request by ButterflyTech**
SYNC content to office commit [d3ace3059c4a92ad4b14b3cd05b5664d5359fb0d](https://github.com/jhipster/jhipster.github.io/commit/d3ace3059c4a92ad4b14b3cd05b5664d5359fb0d), and fixed a lot of chinese translation issues.

I cloned the official website code on September 25th, and spent a month on the interpret to translate most of the content into Chinese. (There are too many change logs. I may need more time to translate later. I think the change log is very It's important to let everyone know the history of the entire Jhipster project) and improve the quality of the previous translation. If my submission is unreasonable, please feel free to contact me. I will continue to contribute this project in the future and share this excellent project with more people.

### CODE REVIEW

**Comment by deepu105** [comment]
@jhipstercn can you check this when you have time

File: _config.yml [hunk 1]
```diff
@@ -8,8 +8,10 @@ kramdown:
   input: GFM
 gems:
   - jekyll-redirect-from
+  - jemoji
```
Comment (deepu105) [inline _config.yml]: Why is this needed?

File: _layouts/default.html [hunk 2]
```diff
@@ -436,22 +424,14 @@ <h5 class="text-left"><i class="fa fa-database fa-fw"></i> <a class="link" href=
...
-                                    </div>
                                 </div>
                             </div>
                         </div>
                         <div class="footer-below">
                             <div class="container">
                                 <div class="row">
                                     <div class="col-lg-12">
-                                        Copyright &copy; JHipster 2013-2019 | Theme based on <a href="https://github.com/IronSummitMedia/startbootstrap-freelancer" target="_blank">Freelancer</a> and <a href="https://github.com/tui2tone/flat-admin-bootstrap-templates" target="_blank">Flat admin</a>
+                                        Copyright &copy; JHipster 2013-2019 | Theme based on <a href="https://github.com/IronSummitMedia/startbootstrap-freelancer" target="_blank">Freelancer</a> and <a href="https://github.com/dulumao/flat-admin-bootstrap-templates" target="_blank">Flat admin</a>
```
Comment (deepu105) [inline _layouts/default.html]: Good catch. @PierreBesson seems like someone changed the URL here as the original was `https://github.com/dulumao/flat-admin-bootstrap-templates`. Hope it wasn't with some malicious intent

**Comment by deepu105** [comment]
Thanks @ButterflyTech

**Comment by PierreBesson** [comment]
Thank a lot @ButterflyTech. I know this takes a long time to do.

File: _config.yml (hunk 1 above)
Comment (ButterflyTech) [reply _config.yml]: because of this file: https://raw.githubusercontent.com/jhipster/jhipster.github.io/master/_config.yml
 
I translate the repo base on [jhipster.github.io](https://github.com/jhipster/jhipster.github.io), so I merged this file.
